src/main/charts/confluence/templates/configmap-fluentd.yaml

```diff
@@ -0,0 +1,40 @@
+{{ if .Values.fluentd.enabled }}
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Here we define a `ConfigMap` which contains the `fluentd.conf` file. This in turn is injected into the pods as a mounted volume.